### PR TITLE
docs: update airgap self-hosted pages

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR updates VerteX and Palette self-hosted airgap pages:
- It removes repeated steps that were featured on both the Setup and Install pages.
- It provides clarification on release-specific and generic OVAs on the Setup page.
- It adds the Setup guide as a prereq for the Install guide.
- It includes the Palette installation binary as a prereq for the Setup guide and includes the steps to execute the bin.

## Changed Pages

💻 [Self-hosted Env Setup](https://deploy-preview-4131--docs-spectrocloud.netlify.app/enterprise-version/install-palette/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions/)
💻 [Self-hosted Install](https://deploy-preview-4131--docs-spectrocloud.netlify.app/enterprise-version/install-palette/install-on-vmware/airgap-install/install/)
💻 [VerteX Env Setup](https://deploy-preview-4131--docs-spectrocloud.netlify.app/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions/)
💻 [VerteX Install](https://deploy-preview-4131--docs-spectrocloud.netlify.app/vertex/install-palette-vertex/install-on-vmware/airgap-install/install/)

## Jira Tickets

🎫 [DOC-1371](https://spectrocloud.atlassian.net/browse/DOC-1371)

## Backports

Can this PR be backported?

- [ x ] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1371]: https://spectrocloud.atlassian.net/browse/DOC-1371?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/install.md [hunk 1]
```diff
@@ -113,54 +106,45 @@ playback speed.
...
-    guide for more information about deploying an OVA in vCenter.
+    Alternatively, you can use any Linux x86-64 system that has the Docker daemon and Palette CLI installed, as well as
+    connectivity to the VMware vSphere environment where Palette VerteX will be deployed.
 
     :::
 
-5.  Open a terminal window and invoke the Palette CLI by using the `ec` command to install the enterprise cluster. The
-    interactive CLI prompts you for configuration details and then initiates the installation. For more information
-    about the `ec` subcommand, refer to [Palette Commands](../../../../automation/palette-cli/commands/ec.md).
+2.  Invoke the Palette CLI by using the `ec` command to install the enterprise cluster. The interactive CLI prompts you
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/install.md:129]: I think we need to mention the issues with the Palette CLI existing in a different location when using the generic OVA.

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/install.md [hunk 2]
```diff
@@ -88,47 +83,41 @@ playback speed.
...
 
-1.  Log in to your vCenter environment.
+1.  In your terminal, use the following command template to SSH into the Palette airgap support VM. Enter the path to
+    your private SSH key, your username, and the IP or domain of the airgap support VM. The default username is
+    `ubuntu`.
 
-2.  Create a vSphere VM and Template folder with the name `spectro-templates`. Ensure this folder is accessible by the
-    user account you will use to deploy the airgap Palette installation.
+    ```shell
+    ssh -i </path/to/private/key> ubuntu@<vm-ip-or-domain>
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/install.md:91]: `identity_file` is the title of the flag -i, not its long form. There is no longform for this flag :D

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md [hunk 3]
```diff
@@ -349,9 +353,37 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
...
+    Uncompressing Airgap Setup - Version 4.4.14  100%
+    Setting up CLI
+    Setting up Manifests
+    Setting up Packs
+    ...
+
+    Setup Completed
+    ```
+
+    Once the airgap binary is executed, you will receive a **Setup Completed** success message.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md:383]: Ops. Thank you! :)

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md [hunk 4]
```diff
@@ -353,9 +357,38 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
...
+    Uncompressing Airgap Setup - Version 4.4.14  100%
+    Setting up CLI
+    Setting up Manifests
+    Setting up Packs
+    ...
+
+    Setup Completed
+    ```
+
+    Once the Palette VerteX airgap binary is executed, you will receive a **Setup Completed** success message.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md:388]: :white_check_mark:

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/install.md [hunk 5]
```diff
@@ -88,47 +83,41 @@ playback speed.
...
-    guide for more information about deploying an OVA in vCenter.
+    Alternatively, you can use any Linux x86-64 system that has the Docker daemon and Palette CLI installed, as well as
+    connectivity to the VMware vSphere environment where Palette will be deployed.
 
     :::
 
-5.  Open a terminal window and invoke the Palette CLI by using the `ec` command to install the enterprise cluster. The
-    interactive CLI prompts you for configuration details and then initiates the installation. For more information
-    about the `ec` subcommand, refer to [Palette Commands](../../../../automation/palette-cli/commands/commands.md).
+2.  Invoke the Palette CLI by using the `ec` command to install the enterprise cluster. The interactive CLI prompts you
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/install.md:101]: Yes, that's true! Done :)

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/install.md (hunk 1 above)
Comment (caroldelwing) [reply docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/install.md:123]: :white_check_mark:

**Pull request merged**